echopype/utils/misc.py

```diff
@@ -11,3 +14,43 @@ def camelcase2snakecase(camel_case_str):
             camel_case_str = camel_case_str[:i] + "_" + camel_case_str[i:]
 
     return camel_case_str.lower()
+
+
+def depth_from_pressure(pressure, latitude=30, atm_pres_surf=0):
+    """
+    Convert pressure to depth using UNESCO equation.
+    Parameters
+    ----------
+    pressure : array_like
+        Pressure in dbar
+    latitude : float
+        Latitude in degrees
+    atm_pres_surf : float
+        Atmospheric pressure at the surface in dbar
+    Returns
+    -------
+    depth : array_like
+        Depth in meters
+    """
+    # Constants
+    g = 9.780318
+    k1 = 1.82e-15
+    k2 = 2.279e-10
+    k3 = 2.2512e-5
+    k4 = 9.72659
+    k5 = 5.2788e-3
+    k6 = 2.36e-5
+    k7 = 1.092e-6
+
+    # Calculate depth
+    x = np.sin(np.deg2rad(latitude))
+    pressure = np.subtract(pressure, atm_pres_surf)
+    depth_w_g = (
+        -k1 * np.power(pressure, 4)
+        + k2 * np.power(pressure, 3)
+        - k3 * np.power(pressure, 2)
+        + k4 * pressure
+    )
+    gravity = g * (1.0 + k5 * np.power(x, 2) + k6 * np.power(x, 4)) + k7 * pressure
```

Comment:
A few style suggestions, for readability:
- You can use the `-` operator with numpy arrays and scalars, instead of `np.subtract`
- The `**` operator works equally well as `np.power`, but is easier to read
- Moved `x` declaration to just before `gravity`, where it's first used

```suggestion
    pressure = pressure - atm_pres_surf
    depth_w_g = -k1 * pressure**4 + k2 * pressure**3 - k3 * pressure**2 + k4 * pressure
    x = np.sin(np.deg2rad(latitude))
    gravity = g * (1.0 + k5 * x**2 + k6 * x**4) + k7 * pressure
```

Reply:
Updated, removed the usage of `np.power` and `np.subtract` and also moved `x` declaration just before `gravity` calculation!